packages/cli/src/commands/server/copyToClipBoard.ts

```diff
@@ -13,7 +13,11 @@ import path from 'path';
 import fs from 'fs';
 
 const xsel = path.join(__dirname, 'external/xsel');
-fs.chmodSync(xsel, '0755');
+try {
+  fs.chmodSync(xsel, '0755');
+} catch (e) {
+  console.warn(`Failed to chmod xsel: ${e.message}`);
```

Comment:
Can you please use `Logger.warn`? It's used throughout the repository, so you can check how to import it